### ISSUE TITLE
TypeError: main() got an unexpected keyword argument 'verbose'

### ISSUE THREAD

**Issue by krystofbe** [src-r-r/python-stanza#6]
calling stanza always returns 

```
TypeError: main() got an unexpected keyword argument 'verbose'
```

**Comment by krystofbe** [comment]
After removing the verbose command from the source code i get another error

```
stanza --dev-requirements requirements.txt
/Users/krystof/.local/pipx/shared/lib/python3.10/site-packages/_distutils_hack/__init__.py:17: UserWarning: Distutils was imported before Setuptools, but importing Setuptools also replaces the `distutils` module in `sys.modules`. This may lead to undesirable behaviors or errors. To avoid these issues, avoid using distutils directly, ensure that setuptools is installed in the traditional way (e.g. not an editable install), and/or make sure that setuptools is always imported before distutils.
  warnings.warn(
/Users/krystof/.local/pipx/shared/lib/python3.10/site-packages/_distutils_hack/__init__.py:30: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
Traceback (most recent call last):
  File "/Users/krystof/.local/bin/stanza", line 8, in <module>
    sys.exit(main())
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/stanza/cli.py", line 42, in main
    convert_command(requirements, dev_requirements, path, name, version)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/stanza/lib/core.py", line 248, in convert_command
    converter.write_toml(base_dir)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/stanza/lib/core.py", line 207, in write_toml
    content = self.get_toml_content(base_dir)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/stanza/lib/core.py", line 200, in get_toml_content
    layout = Layout(self.project.name, **self._layout_kwargs)
  File "/Users/krystof/.local/pipx/venvs/stanza/lib/python3.10/site-packages/stanza/lib/core.py", line 186, in _layout_kwargs
    "description": self._setup_data["description"],
KeyError: 'description'
```

am i doing something wrong?

**Comment by src-r-r** [comment]
No. That's what's known in the industry as a "bug."

Thank you for your report.

Should be a pretty simple fix. You happen to have the setup file that was used? Feel free to replace any sensitive information.

**Comment by src-r-r** [comment]
No. That's what's known in the industry as a "bug."

Should be a pretty simple fix. You happen to have the setup file that was used? Feel free to replace any sensitive information.

**Comment by krystofbe** [comment]
this is the current `setup.py` file

```
import os
from setuptools import setup


with open(os.getenv('REQUIREMENTS_IN', 'requirements.in')) as f:
    requirements = [
        req.rsplit('#egg=', 1)[1] if req.startswith('-e') else req
        for req in f.read().splitlines()]


setup(
    name='cuthbert',
    version='0.1',
    packages=['cuthbert'],
    install_requires=requirements,
)

```

**Comment by src-r-r** [comment]
Thank you for reporting this issue, @krystofbe !
It's now fixed with the PR #7
